wiki/Help_Centre/id.md

```diff
@@ -1,1010 +1,81 @@
-
-Pastikan untuk menyertakan chatlog atau bukti lain di email, dan staf pendukung kami akan memeriksa kasus Anda dan melihatnya untuk Anda.
-
-#### Apa ini "laman pengguna" yang saya lihat di profil pemain lain?
-
-**Laman pengguna adalah elemen profil khusus yang dapat diakses oleh semua pemain dengan [supporter tag](https://osu.ppy.sh/home/support). Laman ini dapat diisi dengan apa pun yang Anda suka - foto pencapaian Anda, beatmap Anda, pada dasarnya apa pun yang Anda inginkan!**
-
-Pemain yang memiliki [supporter tag](https://osu.ppy/sh/home/support)​ diberikan hak istimewa untuk menyunting laman pengguna mereka sendiri untuk ditampilkan di profil mereka supaya semua orang dapat melihatnya. Laman pengguna dapat diisi denagn apa pun yang Anda suka - selama mengikuti ​[aturan komunitas​](/wiki/Rules)!
-
-Bahkan jika ​[supporter tag](https://osu/ppy/sh/home/support)​ Anda sudah habis, laman pengguna Anda akan ​tetap terlihat​ dan Anda masih akan dapat menyunting isinya.
-
-#### Apakah saya perlu menggunakan alamat email saya untuk mendaftar?
-
-**Tidak. Kami mengautentikasi akun Anda dengan meminta Anda masuk melalui klien game, jadi email tidak diperlukan selama proses pendaftaran.**
-
-Jika Anda mengalami kesulitan untuk terhubung melalui klien game, periksa pengaturan firewall dan antivirus Anda untuk memastikan osu! tidak diblokir.
-
-Meskipun tidak memerlukan email, kami menyarankan Anda mengaitkan satu email dengan akun Anda, andaikan Anda kehilangan kata sandi atau tidak dapat mengakses akun Anda, email pendaftaran Anda adalah titik kontak pertama kami dengan Anda.
-
-#### Mengapa saya tidak dapat mencari artis beatmap yang namanya kurang dari 3 karakter?
-
-**Mesin pencari lokal kami tidak dapat menangani ini karena keterbatasan teknis, tetapi Anda masih dapat mencari menggunakan bilah Google Custom Search yang terletak di bagian kiri atas sebagian besar laman situs web.**
-
-Cukup ketikkan nama artis yang Anda cari ke Google Custom Search bar dan biarkan Google akan melakukan sisanya.
-
-#### Mengapa statistik di laman profil pengguna saya salah?
-
-**Anda mungkin melihat statistik Anda untuk game mode lain! Situs web secara default menampilkan skor Anda untuk mode yang terakhir Anda mainkan.**
-
-Jika Anda tidak aktif selama jangka waktu yang panjang dan statistik Anda ditampilkan salah, mainkan beberapa map untuk menampilkannya lagi. Pengguna yang tidak aktif dapat menyembunyikan skor mereka untuk menjaga peringkat tetap fresh.
-
----
-
-## Instalasi dan Registrasi
-
-### Instalasi
+2. Klik menu `Options` yang ada pada menu utama, atau tekan `CTRL` + `O`.
+3. Ketik `release` pada kolom pencarian (*quick-search bar*) yang muncul. Kamu akan melihat versi osu! yang sedang kamu gunakan
+4. Pastikan kamu sedang menggunakan osu! versi `Cutting Edge (Experimental)`.
+5. Apabila pada langkah sebelumnya kamu mengganti versi osu! yang kamu gunakan, klik tombol *restart* yang muncul pada sisi bawah layar.
```

Comment:
*restart*?

![Screenshot from 2021-06-27 15-41-07](https://user-images.githubusercontent.com/4964985/123538254-3ae64800-d75e-11eb-987b-84be3396d48e.png)

![Screenshot from 2021-06-27 15-41-14](https://user-images.githubusercontent.com/4964985/123538260-3e79cf00-d75e-11eb-8828-3f410cef732e.png)

Reply:
oh, ini kalimat bahasa inggris aslinya itu memang `Click the restart button that appears at the bottom` btw kak, nggak ada kata-kata spesifiknya :(